### DISCUSSION TITLE
스터디 - 9주차 목록

### DISCUSSION THREAD

**Discussion by vi2920va** [vi2920va/FE_interview#87] [General]
> 📢기존 팀원이 했던 거 리팩토링을 하고 싶은 경우 제목 옆에 추가 또는 리팩토링 이라고 적어주세요.
🏥 회고록 작성은 없애고, 말로 하는 걸로 변경되었습니다.

## 시간표
2022.01.06(목) - 오후 9시
21:00 ~ 21:15 팀원들 PR 남긴 거 리뷰 시간
21:15 ~ 21:35 모의 면접,1:1, 1:다로 모의면접 진행(상대방이 공부한 것을 기준으로 2개 이상 질문 진행)
21:35 ~ 21:50 모의 면접 진행이 끝난 후, 토론 장소에 모여서 다른 분들에게 궁금했던 걸 토론을 진행
21:50 ~ 22:00 프론트엔드 모의 면접 질문 정리에 작성 후, 돌아가면서 소감 한마디 하고 스터디 종료 

## 송현
- React - props.children(연속해서 쓸 경우와, 예제 정리)
- JavaScript - 변수(리팩토링)

## 영생
- TypeScript - 제네릭(이어서 작업)

## 동기
- JavaScript - 타이머, 디바운스와 스로틀

## 두현
- JavaScript - Webpack(리팩토링)
- JavaScript - Prototype

## 수민
- JavaScript - Lexical scope vs Dynamic scope

## 남은 
- JavaScript - 데이터타입

**Comment by ki7348** [comment]
동기
* JavaScript - 타이머, 디바운스와 스로틀

**Comment by NAMU1105** [comment]
남은

* JavaScript - 데이터타입

**Comment by ttumzzi** [comment]
수민
- JavaScript - Lexical scope vs Dynamic scope

**Comment by whdudtod1273** [comment]
TypeScript - 제네릭 이어서
